.travis.yml

```diff
@@ -44,6 +44,12 @@ script:
   - |
     if [ "$TRAVIS_OS_NAME" = linux ]; then
       export TEST_DEBUG=1
+      docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+      docker tag $CONTROLLER_IMAGE ${CONTROLLER_IMAGE_NAME}:test
```

Comment:
I would tag this with the travis build number as well, that way builds won't stomp over each other